src/applications/proxy-rewrite/local-proxy-rewrite.js

```diff
@@ -60,7 +60,7 @@ function fallbackToTeamSiteServer(buildOptions) {
 }
 
 function setupLocalProxyRewrite(devServer, buildOptions) {
-  devServer.use(injectLocalBundle(buildOptions));
+  devServer.use(injectLocalBundle());
```

Comment:
`buildOptions` was removed from the `injectLocalBundle()` function. It doesn't seem to be used inside the function